src/Input/Textarea/Textarea.tsx

```diff
@@ -9,133 +9,117 @@ import {
   TextareaLabel,
 } from '../../Style/Input/TextareaStyle';
 
-class Textarea extends React.PureComponent<Props, State> {
-  textareaInputRef: React.RefObject<HTMLTextAreaElement>;
-
-  state = {
-    floating: false,
-    rows: 4,
-    minRows: 4,
-    maxRows: 12,
-    textareaMaxHeight: 0,
-  };
-
-  constructor(props: Props) {
-    super(props);
-    this.textareaInputRef = props.forwardedRef || React.createRef();
-  }
-
-  handleFocusChange = (
-    onBlur: (e: React.FocusEvent<HTMLTextAreaElement>) => void
-  ) => {
-    const listener = (e: React.FocusEvent<HTMLTextAreaElement>) => {
-      this.setState({
-        floating: e.target.value.length > 0,
-      });
-
-      if (onBlur !== undefined) {
-        return onBlur(e);
-      }
-    };
+const MIN_ROWS = 4;
+const MAX_ROWS = 12;
 
-    return listener;
-  };
+const Textarea: React.FunctionComponent<Props> = props => {
+  const {
+    label,
+    value,
+    status,
+    disabled,
+    className,
+    onBlur,
+    onChange,
+    removeFloatingLabel,
+    ...restProps
+  } = props;
 
-  handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    const { minRows, maxRows } = this.state;
+  const [floating, setFloating] = React.useState<boolean>(false);
+  const [rows, setRows] = React.useState<number>(4);
```

Comment:
can use `MIN_ROWS` to instead of `4`

Reply:
Updated!

Reply:
Oh @westwood846, seems you miss this change. 🙏

Reply:
Whoops :D Fixed.